src/templates/components/navigation.html

```diff
@@ -27,14 +27,14 @@
                 <li class="dropdown">
                     <a href="https://www.fedramp.gov/participate/3paos/" title="Assessors">Assessors</a>
                     <ul class="dropdown-content right">
-                        <li><a href="https://www.fedramp.gov/participate/3paos/a2la-accreditation/" title="A2LA Accreditation">A2LA Accreditation</a></li>
+                        <li><a href="https://www.fedramp.gov/participate/assessors/a2la-accreditation/" title="A2LA Accreditation">A2LA Accreditation</a></li>
                     </ul>
                 </li>
                 <li><a href="https://www.fedramp.gov/provide-public-comment/" title="Provide Public Comment">Provide Public Comment</a></li>
             </ul>
         </li>
         <li class="dropdown">
-            <a href="#"
+            <a href="javascript:;"
```

Comment:
Ah-ha, this makes sense.  Curious about `javascript:;` - can it be an empty `href` instead?  Not a blocker/breaker in either case, though.

Reference:
https://docs.angularjs.org/api/ng/directive/a